### DISCUSSION TITLE
Hy for data science

### DISCUSSION THREAD

**Discussion by ktodyruik** [hylang/hy#2491] [Q&A] [answered]
Hi,

Is anyone using hy for data science? What works well? What's your setup? I tried jupyter (calysto hy) a while back and didn't have much success. I was wondering what's working well for others in this space? Any tips or feedback would be very appreciated!

Thanks,

**Comment by Kodiologist** [comment, marked as answer]
I always thought "data science" was a silly term, since all science involves data; it seems to just be a sexed-up equivalent of "data analysis", with the insinuation that the person described is more skilled as a programmer than a statistician. Anyway, I'm a data analyst by profession and I use Hy for it most of the time, sometimes with Org-mode and Emacs's Inferior Lisp mode. I keep some Hy-related support code for Emacs and Org in [Daylight](https://github.com/Kodiologist/Daylight). You can see some examples in source packages and notebooks for [some of my recent research projects](http://arfer.net/projects), although the code isn't always easy to get running because it's more for my own use than public consumption.
